### ISSUE TITLE
Flux controls

### ISSUE THREAD

**Issue by bikedel** [ragnarlotus/vue-flux#107]
Hello

I am following documentation for v6. Has the implementation of the controls changed  in v7?

**Comment by ragnarlotus** [comment]
Not really, but check https://github.com/ragnarlotus/vue-flux/blob/v7/src/App.vue example about how to implement it
